specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -1131,12 +1131,12 @@
       "properties": {
         "id": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
           "description": "Azure resource Id of the VM."
         },
         "uuid": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
```

Comment:
May I ask why this property is writeable? Per my understanding, service side is able to get the vm info via the parameter VmNameParameter while creating this resource, right?